### PR TITLE
New Button Component Use Cases 

### PR DESCRIPTION
**Pull request by chirag-singhal**
## Summary
 * New use cases for button component in rn-tester
 * E2E tests for the button component

## Changelog
[General] [Changed] - Changed use cases for button component in rn-tester

## Test Plan 
image
After - 
image
Before -
image

### CODE REVIEW

**Comment by chirag-singhal** [comment]
@rickhanlonii  I have created all my four PRs. Please have a look. Thanks

**Review by ermanzohre — APPROVED** [review]

File: packages/rn-tester/js/examples/Button/ButtonExample.js [hunk 1]
```diff
@@ -28,93 +27,115 @@ exports.description = 'Simple React Native button component.';
...
-    title: 'Adjusted color',
-    description: ('Adjusts the color in a way that looks standard on each ' +
-      'platform. On iOS, the color prop controls the color of the text. On ' +
+    title: 'Button with color="red"',
+    description: ('Note: On iOS, the color prop controls the color of the text. On ' +
       'Android, the color adjusts the background color of the button.': string),
     render: function(): React.Node {
       return (
-        <RNTesterThemeContext.Consumer>
-          {theme => {
```
Comment (rickhanlonii) [inline packages/rn-tester/js/examples/Button/ButtonExample.js:61]: The old examples used a theme context to support dark theme, is that not supported here?

**Comment by rickhanlonii** [comment]
Also, can you include a screenshot or demo in the test plan?

File: packages/rn-tester/js/examples/Button/ButtonExample.js (hunk 1 above)
Comment (chirag-singhal) [reply packages/rn-tester/js/examples/Button/ButtonExample.js:61]: Yes @rickhanlonii

**Comment by chirag-singhal** [comment]
Updated the test plan!

**Review by ermanzohre — APPROVED** [review]

File: packages/rn-tester/js/examples/Button/ButtonExample.js (hunk 1 above)
Comment (rickhanlonii) [reply packages/rn-tester/js/examples/Button/ButtonExample.js:61]: You mean yes it's not supported? Can we support it?

Comment (chirag-singhal) [reply packages/rn-tester/js/examples/Button/ButtonExample.js:61]: Sure Ricky

Comment (chirag-singhal) [reply packages/rn-tester/js/examples/Button/ButtonExample.js:61]: Done!

**Comment by rickhanlonii** [comment]
@chirag-singhal is it correct to say that even though the theme code is there, there's no dark mode in the UI changes?

**Comment by chirag-singhal** [comment]
Yes @rickhanlonii, some changes are needed  in the UI components to use colors from theme context just like this component

**Comment by chirag-singhal** [comment]
Should i do those changes in this PR?

**Comment by rickhanlonii** [comment]
@chirag-singhal no, that would be separate work

**Review by rickhanlonii — APPROVED** [review]

**Review by ermanzohre — APPROVED** [review]

**Pull request closed without merging**